search-resources-api/src/main/java/no/unit/nva/search/SearchResourcesApiHandler.java

```diff
@@ -68,4 +77,16 @@ protected SearchResourcesResponse processInput(Void input,
     protected Integer getSuccessStatusCode(Void input, SearchResourcesResponse output) {
         return HttpStatus.SC_OK;
     }
+
+    @Override
+    protected void writeOutput(Void input, SearchResourcesResponse output)
```

Comment:
we do not need to overwrite anything now

Reply:
Ups, forgot this one...